src/wagtail_live/webapp/receiver.py

```diff
@@ -41,7 +43,33 @@ def get_message_text(self, message):
     def get_message_files(self, message):
         """See base class."""
 
-        return message["files"] if "files" in message else []
+        return message["images"]
+
+    def get_image_title(self, image):
+        """See base class."""
+
+        return image["image"]["name"].replace("-", " ")
+
+    def get_image_name(self, image):
+        """See base class."""
+
+        return image["image"]["name"]
+
+    def get_image_mimetype(self, image):
+        """See base class."""
+
+        mime_type = image["image"]["name"].split(".")[-1]
+        return "jpeg" if mime_type == "jpg" else mime_type
+
+    def get_image_dimensions(self, image):
+        """See base class."""
+
+        return (image["image"]["width"], image["image"]["height"])
+
+    def get_image_content(self, image):
+        """See base class."""
+
+        return Image.objects.get(id=image["id"]).image
```

Comment:
Won't this give a problem with the Delete view you also added below? It seems to only delete the image, but this view might raise an Image.DoesNotExist then? Or is it never called anymore?

Reply:
Sorry @lucasmoeskops I tried hard but I can't get your point.
Is it the DeleteImageView that you are talking about?
If so, what problem might occur and when?

Reply:
Yes, in the DeleteImageView, the only thing that seems to be done is to delete the specified Image instance. But can the image not still be used in an existing message, so that this code will raise an error when trying to do `.get(id=image["id"])`? 

I mean basically, the image seems to be deleted, but the message seems to still exist. Trying to reproduce it, but having some issues with csrf token validation at the moment interestingly :-(

Reply:
I expect that the user will click "Save changes" in which case the image will no longer be in the message. If we send an update, the id of the message deleted won't be in the update so I think we should not have an error there.

I get sometimes a csrf token error but when I refresh or close and reopen the tab it works fine. Maybe there is something else too.

Reply:
You are right! It is no issue because a different message is sent. All right :-) Then I'll merge the request!